com.unity.render-pipelines.universal/Runtime/2D/Passes/Render2DLightingPass.cs

```diff
@@ -213,7 +213,7 @@ private void Render(ScriptableRenderContext context, CommandBuffer cmd, ref Rend
                     if (layerBatch.lightStats.totalNormalMapUsage > 0)
                     {
                         filterSettings.sortingLayerRange = layerBatch.layerRange;
-                        var depthTarget = m_NeedsDepth ? depthAttachmentHandle.nameID : BuiltinRenderTextureType.None;
+                        var depthTarget = false ? depthAttachmentHandle.nameID : BuiltinRenderTextureType.None;
```

Comment:
not sure if this is a temporary change for testing purpose or final. If final you can remove the ternary operator completely and replace it with BuiltinRenderTextureType.None